File: Python/SkillsDemo.py
count = 0

while count < 5:  
    fName = (input("Enter First Name: "))
    lName = (input("Enter Last Name: ")) 

    grade1 = float(input("Enter Grade: "))
    if grade1 < 0 or grade1 > 100:  
        print("Invaild Grade. Enter vaild grade 0 - 100")
        grade1 = float(input("Enter Grade: "))

    grade2 = float(input("Enter Grade: "))
    if grade2 < 0 or grade2 > 100:
        print("Invaild Grade. Enter vaild grade 0 - 100")
        grade2 = float(input("Enter Grade: "))

    grade3 = float(input("Enter Grade: "))
    if grade3 < 0 or grade3 > 100:
        print("Invaild Grade. Enter vaild grade 0 - 100")
        grade3 = float(input("Enter Grade: "))

    total = (grade1 + grade2 + grade3)
    avg = total/3 

    print("\nYour Initials are: "+ fName[0] + ", " + lName[0])
    print("\nYour Average Grade is: ", round(avg, 2),"%")
    print("\n")
    count += 1 

File: Java/SkillsDemo.java
package Java;

import java.util.Scanner;


	class SkillsDemo {

		public static void main(String[]args) {

			String FName, LName;
			char Fn, Ln;
			int num1, num2, num3, Total;
			double Average;

				Scanner MyInput = new Scanner(System.in);


			System.out.println("Type First Name: ");
			FName = MyInput.next();
			Fn = FName.charAt(0);


			System.out.println("Type Last Name: ");
			LName = MyInput.next();
			Ln = LName.charAt(0);


			System.out.println("Type Number: ");
			num1 = MyInput.nextInt();
			System.out.println("Type Number: ");
			num2 = MyInput.nextInt();
			System.out.println("Type Number: ");
			num3 = MyInput.nextInt();

			System.out.println("*****************");
			System.out.println("*****************");

			System.out.println( Fn+ "," +Ln);
			System.out.println("Your Numbers: " +num1+"," +num2+ ","  +num3+" ");

				Total = num1 + num2 + num3;

			System.out.println( "The Total is: " +Total);

				Average = (Total)/3;


			System.out.println("The Average is: " +Average);

        MyInput.close();





		}}

        